README.md

```diff
@@ -87,6 +87,45 @@ Full documentation can be found on the [Lisk documentation site][].
 | [@liskhq/lisk-passphrase](packages/lisk-passphrase)     |  [![](https://img.shields.io/badge/npm-v1.1.0-green.svg)](https://www.npmjs.com/package/@liskhq/lisk-passphrase)  | Mnemonic passphrase helpers for use with Lisk-related software     |
 | [@liskhq/lisk-transactions](packages/lisk-transactions) | [![](https://img.shields.io/badge/npm-v1.1.0-green.svg)](https://www.npmjs.com/package/@liskhq/lisk-transactions) | Everything related to transactions according to the Lisk protocol  |
 
+## Tests
+
+To run tests for all packages in lisk-elements, run the following command in the root folder:
+
+```
+npm test
+```
+
+To run tests for a specific package, run the same command in the relevant package directory.
+
+```
+cd packages/lisk-cryptography
```

Comment:
package name should be more generic?

Reply:
would prefer an actual package, and using `packages/lisk-elements` as an example here may be confusing.  maybe after the instruction add "Example:"?

Reply:
yea i think that would be great